### DISCUSSION TITLE
Arm64 Docker Image?

### DISCUSSION THREAD

**Discussion by estubmo** [postgresml/pgcat#708] [Ideas]
Would it be possible to add building and publishing of an arm64 image to the [Build and Push-workflow](https://github.com/postgresml/pgcat/actions/workflows/build-and-push.yaml) along with the [amd64 image](https://github.com/postgresml/pgcat/pkgs/container/pgcat)?

**Comment by drdrsh** [comment]
This is done 
https://github.com/postgresml/pgcat/pull/774
